Example/Models/UserProfileModel.swift

```diff
@@ -3,15 +3,23 @@ import UIKit
 struct UserProfileModel: Codable {
     var displayName: String?
     var profileImageURI: String?
-    var contactList: [String]?
+    var contactList: [Contact]?
 
-    init(displayName: String, profileImageURI: String?, contactList: [String]?) {
+    init(displayName: String, profileImageURI: String?, contactList: [Contact]?) {
         self.displayName = displayName
         self.profileImageURI = profileImageURI
         self.contactList = contactList
     }
 }
 
+struct Contact: Codable {
+    var id: String
+
+    init(id: String) {
+        self.id = id
+    }
+}
+
```

Comment:
We can use the public class `MAContact` right ? This seems a duplicate just for the sample app

Reply:
I don't know why but I cannot import `MAContact`, neither of `MATokenInfo`.

Reply:
I think this should work with `MAContact` - it was working for me when I tried it. 
@minh-rakuten Did you try cleaning the XCode build folder? Also make sure to add `import MiniApp` to this file.

Reply:
Thanks. Re-build in XCode works for me.